players/equipment.cpp

```diff
@@ -257,6 +260,9 @@ bool doWear(const std::shared_ptr<Player>& player, cmd* cmnd) {
     if(!player->ableToDoCommand())
         return(false);
 
+    if(player->isMagicallyHeld(true))
```

Comment:
`wearAll` doesn't seem to have an equivalent check even in `ableToDoCommand`

Reply:
This is ok, actually, because wearAll() is called from wear()....So it checks that first anyway.